docs/sidebars.js

```diff
@@ -128,6 +128,9 @@ const sidebars = {
         'transform/sql',
         'transform/split',
         'transform/json',
+        'transform/replace',
+        'transform/udf',
+        'transform/uuid',
```

Comment:
In my knowledge, the sidebars will be auto generated(I am not sure), but you don't need to do this change in this PR.